builder/bundles.go

```diff
@@ -925,15 +952,77 @@ func fixOSRelease(filename, version string) error {
 	// See also: https://github.com/clearlinux/mixer-tools/issues/113
 
 	var newBuf bytes.Buffer
+	mixInfoFlags := make([]bool, MixInfoCount)
 	scanner := bufio.NewScanner(f)
+
 	for scanner.Scan() {
 		text := scanner.Text()
+		if strings.HasPrefix(text, "NAME=") {
+			text = "NAME=" + "\"" + mixInfo[OSName] + "\""
+			mixInfoFlags[OSName] = true
+		}
 		if strings.HasPrefix(text, "VERSION_ID=") {
-			text = "VERSION_ID=" + version
+			text = "VERSION_ID=" + mixInfo[VersionID]
+			mixInfoFlags[VersionID] = true
+		}
+		if strings.HasPrefix(text, "BUILD_ID=") {
+			text = "BUILD_ID=" + mixInfo[BuildID]
+			mixInfoFlags[BuildID] = true
+		}
+		if strings.HasPrefix(text, "UPSTREAM_URL=") {
```

Comment:
I don't see `UPSTREAM_URL` defined in the os-release documentation, so I would recommend following the guidelines in os-release(5) on naming distro-specific keys:

> Operating system vendors may extend the file format and introduce new fields. It is highly recommended to prefix new fields with an OS specific name in order to avoid name clashes. Applications reading this file must ignore unknown fields. Example: "DEBIAN_BTS="debbugs://bugs.debian.org/""

So, perhaps use `CL_UPSTREAM_URL` instead?

Reply:
I think my point still stands. IMO, we should follow the guidelines in os-release(5) and use a distro-specific prefix to avoid future naming clashes; any generic name, like `UPSTREAM_URL`, has the potential of being adopted as standard in the future.

The arguments against using the `CL_` prefix are valid, so you can just choose a different prefix. Loosening the definition of "distro" to "tool that creates the distro", perhaps use a `MIX_` prefix?